x/protorev/types/constants.go

```diff
@@ -27,6 +27,10 @@ const MaxPoolPointsPerTx uint64 = 50
 // to the maximum execution time (in ms) of protorev per block
 const MaxPoolPointsPerBlock uint64 = 200
 
+// Max number of ticks we can move in a concentrated pool swap. This will be parameterized in a
+// follow up PR.
+const MaxTicksMoved uint64 = 5
```

Comment:
Let's call this MaxTicksCrossed to stay consistent with the CL method naming of `ComputeMaxInAmtGivenMaxTicksCrossed `